gamestonk_terminal/etf/screener_model.py

```diff
@@ -183,7 +183,7 @@ def etf_screener(other_args: List[str]):
 
         export_data(
             ns_parser.export,
-            os.path.dirname(os.path.abspath(__file__)),
+            os.path.dirname(os.path.abspath(__file__)) + "/screeners/",
```

Comment:
Since `>etf` can export screeners or this new stuff. Figured I would break into submenus to keep exports clean

Reply:
Humm, I would agree with something like that if we added the configuration parameters along with the screener data.

Since we don't provide any additional information, I would keep it.

It shows like this:
```
Saved file: /Users/DidierRodriguesLopes/Documents/git/GamestonkTerminal/exports/etf/screeners/etf_screener_20210715_222100.csv
```

Reply:
So if you specify using the flags, all those get added, and your screener file will be a really long yucky thing.  Its just when you provide --config that the save file is not that long (yeah...lazy on my end there since I can just loop through the ini file without having to define anything else)